### DISCUSSION TITLE
MX_MISSING - RSPAMD

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2387] [Help with using Postal]
```
BAYES_HAM (-4.753309) [99.36%]
MX_MISSING (2) [requested record is not found]
FORGED_SENDER (0.5) [info@uhlhosting.ch,nk4d3v@psrp.uhlhosting.ch]
MV_CASE (0.5)
MX_GOOD (-0.01) [psrp.uhlhosting.ch]
```

this is the RSPAMD records, any one knows why I get MX missing? Using API alto to send emails. Is weird because bellow says MX Good, and then MX Missing previous.

**Comment by willpower232** [comment]
it might be a bit of a technicality, psrp.uhlhosting.ch is CNAME'd correctly but technically rp.post.uhlhosting.ch doesn't have an MX record (only an A) so may not be capable of delivering email

**Reply by uhlhosting** [reply]
I believe I added 
<img width="838" alt="image" src="https://user-images.githubusercontent.com/6324047/226610641-509b1795-07a9-4c39-acab-b18143a302a8.png">
them wrong.

**Reply by uhlhosting** [reply]
Added also the missing rp. let's see if this helps.
